### PR TITLE
Qt/Log: display TTY on normal gui log and prepare Qt TTY input (not working yet)

### PR DESCRIPTION
**Pull request by Megamouse**
tty text is set to log level notice

### CODE REVIEW

**Comment by Nekotekina** [comment]
Why? TTY is not just some log. It should also support input.

**Comment by Megamouse** [comment]
sure. I just did what ssshadow asked in chat. If you don't like it I can keep this a minor fix with the file close if that's even an issue

File: rpcs3/rpcs3qt/log_frame.cpp [hunk 1]
```diff
@@ -262,10 +258,18 @@ void log_frame::CreateAndConnectActions()
 	LoadSettings();
 }
 
+log_frame::~log_frame()
+{
+	m_tty_file.close();
```
Comment (Nekotekina) [inline rpcs3/rpcs3qt/log_frame.cpp]: Placebo. You think it doesn't close in fs::file destructor?

File: rpcs3/rpcs3qt/log_frame.cpp [hunk 2]
```diff
@@ -316,24 +348,108 @@ void log_frame::UpdateUI()
...
 
-		// Hackily used the state of the check..  be better if I actually stored this value.
-		if (m_TTYAct->isChecked())
+		if (m_log_tty)
 		{
-			text.chop(1); // remove newline since Qt automatically adds a newline.
+			if (m_show_tty_in_log && ++tty_count == 0)
+			{
+				m_log->setTextColor(tty_col);
+				m_log->append("--- TTY begin ---");
```
Comment (TGEnigma) [inline rpcs3/rpcs3qt/log_frame.cpp]: Consider prefixing the tty log text similarly to log channels

Comment (Megamouse) [reply rpcs3/rpcs3qt/log_frame.cpp]: feel free to pr a better solution. I think this PR is dead anyway

Comment (TGEnigma) [reply rpcs3/rpcs3qt/log_frame.cpp]: I'm pretty sure neko is just occupied and/or doesn't feel like merging prs atm

Comment (Nekotekina) [reply rpcs3/rpcs3qt/log_frame.cpp]: Regarding this PR, I was opposing some TTY changes from the start, so it became a complicated problem.

Comment (TGEnigma) [reply rpcs3/rpcs3qt/log_frame.cpp]: I see. Well, input was added at some point, so it should suffice yes?

Comment (Megamouse) [reply rpcs3/rpcs3qt/log_frame.cpp]: I could change it to just have the input lineedit in tty.
But that will be useless until someone hooks it up

**Comment by Megamouse** [comment]
Closing for inactivity.
the branch will stay online

**Pull request closed without merging**